go.mod

```diff
@@ -10,24 +10,23 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.9.0
-	go.temporal.io/api v1.29.1
-	golang.org/x/sys v0.18.0
+	go.temporal.io/api v1.31.1-0.20240408135840-a6cecb28ed20
```

Comment:
Note: currently the api changes for update admitted event are not in a tagged api release.